components/base/BaseTokenForm.vue

```diff
@@ -8,7 +8,8 @@
 
     <CollectionSelect
       v-model="vSelectedCollection"
-      :collections="collections" />
+      :collections="collections"
+      :showExplainerTest="showExplainerTest" />
```

Comment:
should this be `text`instead of `test`?
```suggestion
      :showExplainerText="showExplainerText" />
```